### PR TITLE
Update usage statement to use the sample name.

### PR DESCRIPTION
**Pull request by RagnarPaulson**

### CODE REVIEW

File: src/QueueProducer/QueueProducer.cs [hunk 1]
```diff
@@ -138,14 +138,14 @@ static void Main(string[] args)
         {
             if (args.Length < 3)
             {
-                Console.WriteLine("Usage: TopicPublisher <host> <username>@<vpnname> <password>");
+                Console.WriteLine("Usage: QueueProduce <host> <username>@<vpnname> <password>");
```
Comment (Mrc0113) [inline src/QueueProducer/QueueProducer.cs]: should be "QueueProducer"

File: src/QueueProducer/QueueProducer.csproj [hunk 2]
```diff
@@ -9,6 +9,6 @@
     <Copyright>Copyright © Solace Corp 2022</Copyright>
```
Comment (Mrc0113) [inline src/QueueProducer/QueueProducer.csproj:9]: should we be changing all copyright dates to 2023? or just leave them as 2022?

Comment (RagnarPaulson) [reply src/QueueProducer/QueueProducer.csproj:9]: As discussed copyright statements is a bigger and separate issue.  There is likely a need to add Copyright lines to every file.

File: src/QueueProducer/QueueProducer.cs (hunk 1 above)
Comment (RagnarPaulson) [reply src/QueueProducer/QueueProducer.cs]: fixed in latest commit.

**Pull request merged**